src/reference/01-General-Info/05-Using-Sonatype.md

```diff
@@ -2,26 +2,89 @@
 out: Using-Sonatype.html
 ---
 
+  [sonatype-ossrhguide]: http://central.sonatype.org/pages/ossrh-guide.html
+  [sonatype-signup]: https://issues.sonatype.org/secure/Signup!default.jspa
+  [sonatype-new-project]: https://issues.sonatype.org/secure/CreateIssue.jspa?issuetype=21&pid=10134
+  [sonatype-requirements]: http://central.sonatype.org/pages/requirements.html
+  [sonatype-coordinates]: http://central.sonatype.org/pages/choosing-your-coordinates.html
+  [sonatype-nexus]: https://oss.sonatype.org/#welcome
+  [sonatype-pgp]: http://central.sonatype.org/pages/working-with-pgp-signatures.html
   [sbt-pgp]: http://scala-sbt.org/sbt-pgp/
+  [sbt-sonatype]: https://github.com/xerial/sbt-sonatype
+  [sbt-release]: https://github.com/sbt/sbt-release
+  [gnupg]: https://www.gnupg.org/
 
-Deploying to Sonatype
+Using Sonatype
 ---------------------
-
 Deploying to sonatype is easy! Just follow these simple steps:
 
+## Sonatype setup
```

Comment:
Probably not necessary to add a heading at the top like this.  Ends up looking like so:

![screen shot 2017-03-30 at 12 55 46 am](https://cloud.githubusercontent.com/assets/358615/24488293/cd2b3190-14e3-11e7-86db-c22ca647eefa.png)